api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/mq/MQCategory13Section1QueriesInterface.java

```diff
@@ -100,22 +100,25 @@ public static final String findPatientsWithLastClinicalConsultationDenominatorB1
 
     public static final String
         findPatientsWithLastClinicalConsultationwhoAreInLinhaAlternativaDenominatorB3 =
-            "Select final.patient_id from ( "
-                + "Select alternativeLine.patient_id,alternativeLine.dataLinha, alternativeLine.ultimaConsulta from ( "
-                + "Select enc.patient_id,enc.encounter_datetime ultimaConsulta,max(obsLinha.obs_datetime) dataLinha from ( "
-                + "Select p.patient_id,max(e.encounter_datetime) encounter_datetime from patient p "
-                + "inner join encounter e on p.patient_id=e.patient_id "
-                + "where p.voided=0 and e.voided=0 and e.encounter_type=6 and "
-                + "e.encounter_datetime >=:startInclusionDate and e.encounter_datetime<=:endRevisionDate  and e.location_id=:location "
+            "select ultimaConsulta.patient_id from ( "
+                + "Select p.patient_id,max(e.encounter_datetime) ultimaConsulta from patient p "
+                + "inner join encounter e on p.patient_id=e.patient_id  "
+                + "where p.voided=0 and e.voided=0 and e.encounter_type=6 and  "
+                + "e.encounter_datetime >=:startInclusionDate and e.encounter_datetime<=:endRevisionDate "
+                + "and e.location_id=:location  "
                 + "group by p.patient_id "
-                + ") enc "
-                + "inner join encounter  e on e.patient_id=enc.patient_id "
+                + ")ultimaConsulta  "
+                + "inner join ( "
+                + "Select p.patient_id, max(obsLinha.obs_datetime) dataLinha, e.encounter_id from patient p "
```

Comment:
@emaposse  In the highlighted code, I see you are returning **e.encounter_id** but 
I don't see the useful of this column .

Reply:
Ok I will remove I put it when I did the query tests.